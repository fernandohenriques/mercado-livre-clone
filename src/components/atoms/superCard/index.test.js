import React from 'react';
import { mount } from 'enzyme';
import { render } from '@testing-library/react';
import SuperCard from './index';

describe('<SuperCard />', () => {
  it('Snapshot testing', () => {
    const { asFragment } = render(
      <SuperCard>
        <div>Awesome content</div>
      </SuperCard>
    );

    expect(asFragment()).toMatchSnapshot();
  });

  it('Should use props className as CSS class', () => {
    const wrapper = mount(
      <SuperCard className="super-card">
        <div>Other awesome content</div>
      </SuperCard>
    );

    expect(wrapper.find('.container').hasClass('super-card')).toBe(true);
  });
});
