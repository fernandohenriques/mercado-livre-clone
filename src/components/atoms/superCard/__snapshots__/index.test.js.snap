// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<SuperCard /> Snapshot testing 1`] = `
<DocumentFragment>
  <div
    class="container"
  >
    <div>
      Awesome content
    </div>
  </div>
</DocumentFragment>
`;
